src/test/kotlin/Utilities/FileIoTest.kt

```diff
@@ -0,0 +1,22 @@
+import Utilities.FileIo
+import kotlin.test.*
+
+class FileIoTest {
+    @Test
+    fun `readFile is able to read from html file`() {
+        val fileIo = FileIo()
+        val expectedResult = "<!doctype>\n" +
+                "<html lang=\"en-US\">\n" +
+                "<head>\n" +
+                "    <title>TODO List</title>\n" +
+                "</head>\n" +
+                "\n" +
+                "<body>\n" +
+                "<main>\n" +
+                "    <strong>Status:</strong> pass\n" +
+                "</main>\n" +
+                "</body>\n" +
+                "</html>\n"
+        assertEquals(expectedResult, fileIo.readResource("test-html.html"))
```

Comment:
This file doesn't have to be an HTML file. It could be any file, and the content could be very simple, e.g., just one line with one word or two lines with one word each. Such a test file would reduce the string concatenation you have to set up in `expectedResult`.